kernel/arch/dreamcast/include/arch/args.h

```diff
@@ -0,0 +1,48 @@
+/* KallistiOS ##version##
+
+   arch/dreamcast/include/arch/args.h
+   (c)2023 Paul Cercueil <paul@crapouillou.net>
+
+*/
+
+/** \file   arch/args.h
+    \brief  Macros for getting argument registers in inline assembly
+
+    This file contains the KOS_ARG(n) macro, which resolves to the register name
+    that corresponds to the nth argument of a function.
```

Comment:
Maybe mention that these are for floating point registers only?

Reply:
I can rename the macro `KOS_FPARG` maybe.